injection/sharedmain/main.go

```diff
@@ -144,7 +156,22 @@ func MainWithConfig(ctx context.Context, component string, cfg *rest.Config, cto
 
 	// Start all of the controllers.
 	logger.Info("Starting controllers...")
-	controller.StartAll(ctx.Done(), controllers...)
+	go controller.StartAll(ctx.Done(), controllers...)
```

Comment:
Why  not run the controllers as part of the error group as well?

Reply:
I guess it would be possible, yeah. Then we could even catch the errors in the controllers. Now we ignore them. Good point.

Reply:
After a discussion with @markusthoemmes we agreed that it's good to leave it as is. The `controller.StartAll` doesn't return any error so there won't be any advantage running it as part of the errgroup.